core/input/keyboard.js

```diff
@@ -126,11 +126,20 @@ export default class Keyboard {
             return;
         }
 
-        // Alt behaves more like AltGraph on macOS, so shuffle the
-        // keys around a bit to make things more sane for the remote
-        // server. This method is used by RealVNC and TigerVNC (and
-        // possibly others).
         if (browser.isMac() || browser.isIOS()) {
+            // If a key is pressed while meta is held down, the keyup will
+            // never be sent in Chrome (bug #108404) and possibly others
+            if (e.metaKey && keysym !== KeyTable.XK_Super_L && keysym !== KeyTable.XK_Super_R) {
+                this._sendKeyEvent(keysym, code, true);
+                this._sendKeyEvent(keysym, code, false);
+                stopEvent(e);
+                return;
+            }
```

Comment:
I think it makes more sense to have this next to the other cases where we don't get a keyup event, further down.

Reply:
It must be before shuffling the keys around, because later in code Super_L will be Alt_R and using that in the condition would be even more confusing, I'd say.

Reply:
Or I could change the condition from `keysym` to `code === 'Meta'`. I try that.